src/main/java/com/buddy/api/domains/adoption/entities/AdoptionPostFollowUpEntity.java

```diff
@@ -38,10 +39,25 @@ public class AdoptionPostFollowUpEntity {
     private UUID followUpId;
 
     @ManyToOne(fetch = FetchType.LAZY, optional = false)
-    @JoinColumn(name = "request_id", referencedColumnName = "request_id", nullable = false)
+    @JoinColumn(
+            name = "request_id",
+            referencedColumnName = "request_id",
+            nullable = false,
+            updatable = false
+    )
```

Comment:
Lembrar de adicionar o Enumerated no enum
```suggestion
  @Enumerated(EnumType.STRING)
  @Column(name = "adaptation_status", nullable = false)
    private AdaptationStatusEnum adaptationStatus;
```